tracer/test/Datadog.Trace.TestHelpers/NativeProcess/NativeMethods.cs

```diff
@@ -0,0 +1,110 @@
+// <copyright file="NativeMethods.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+namespace Datadog.Trace.TestHelpers.NativeProcess;
+
+internal class NativeMethods
+{
+    internal const int DUPLICATE_SAME_ACCESS = 2;
+    internal const int STD_INPUT_HANDLE = -10;
+    internal const int STD_OUTPUT_HANDLE = -11;
+    internal const int STD_ERROR_HANDLE = -12;
+    internal const int STARTF_USESHOWWINDOW = 0x00000001;
+    internal const int STARTF_USESTDHANDLES = 0x00000100;
+    internal const int CREATE_NO_WINDOW = 0x08000000;
+    internal const int CREATE_SUSPENDED = 0x00000004;
+    internal const int CREATE_UNICODE_ENVIRONMENT = 0x00000400;
+    internal const int ERROR_BAD_EXE_FORMAT = 193;
+    internal const int ERROR_EXE_MACHINE_TYPE_MISMATCH = 216;
+
+    [DllImport("kernel32.dll", CharSet = CharSet.Ansi, SetLastError = true, BestFitMapping = false)]
+    internal static extern bool DuplicateHandle(
```

Comment:
Yup, this all looks correct :hide:

Reply:
Actually the CharSet and BestFitMapping are useless here. Good catch.